models/classes/class.QtiTestService.php

```diff
@@ -1246,7 +1246,7 @@ private function verifyItemPermissions(string $json): void
         foreach ($array['testParts'] ?? [] as $testPart) {
             foreach ($testPart['assessmentSections'] ?? [] as $assessmentSection) {
                 foreach ($assessmentSection['sectionParts'] ?? [] as $item) {
-                    if (isset($item['href'], $item['label']) && $item['qti-type'] ?? '' === self::XML_ASSESSMENT_ITEM_REF) {
```

Comment:
this is not necessarily the wrong check
was made to ignore permission checking when importing a test
now when importing a test this method is not used

Reply:
Sorry, I still don't get why the check for not empty `$item['label']` was removed. Can you explain it more detailed?

Reply:
This check was done to bypass the permissions check when importing tests. Now this check is not needed, because now when importing tests this function is not used